wp_serde_helper/src/wp_serde_date.rs

```diff
@@ -18,6 +18,24 @@ pub mod wp_utc_date_format {
         D: Deserializer<'de>,
     {
         let s = String::deserialize(deserializer)?;
+
```

Comment:
These changes warrant some additional scrutiny – I was on the fence about whether to implement a new object for alternative time representations, but the other types I came across are trivially expressible as a UTC datetime so to me it made sense to allow a larger variety of values to be parsed. I added a test for each, but there might be more cases I haven't though of?